bundles/core/org.eclipse.smarthome.core.voice/src/main/java/org/eclipse/smarthome/core/voice/internal/VoiceConsoleCommandExtension.java

```diff
@@ -76,10 +80,21 @@ public void execute(String[] args, Console console) {
                     }
                     return;
                 case SUBCMD_VOICES:
-                    for (Voice voice : voiceManager.getAllVoices()) {
-                        console.println(
-                                voice.getUID() + " " + voice.getLabel() + " - " + voice.getLocale().getDisplayName());
+                    Locale locale = localeProvider.getLocale();
+                    Voice v = voiceManager.getDefaultVoice();
+                    if (v == null) {
+                        TTSService tts = voiceManager.getTTS();
+                        if (tts != null) {
+                            v = voiceManager.getPreferredVoice(tts.getAvailableVoices());
+                        }
                     }
+                    final Voice defaultVoice = v;
+
+                    VoiceHelper.withSortedVoices(voiceManager.getTTSs().stream(), locale, (ttsService, voice) -> {
+                        console.println(String.format("%s%s - %s - %s (%s)", voice.equals(defaultVoice) ? "*" : "",
```

Comment:
Just nit picking: To align the output and make it more human readable we maybe should prepend `"* "` (asterisks followed by whitespace) for the default voice and `"  "` (two whitespaces) else.

Reply:
The nit picking is good because I thought the same when seeing the final result. ;-)